src/public/js/vendor/inspectlet.ts

```diff
@@ -0,0 +1,34 @@
+export {};
+const wid = 812310448;
```

Comment:
Please put this in config file, pass it down from server to client to determine whether Inspectlet should be included or not.